### PR TITLE
[CSL-2273] Add section to disable TLS for swagger commands

### PR DESCRIPTION
**Pull request by akegalj**

### CODE REVIEW

File: wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs [hunk 1]
```diff
@@ -429,6 +429,15 @@ This means that _omitting_ the version number would call the old version of the
 
 Compatibility between major versions is not _guaranteed_, i.e. the request & response formats might differ.
 
+## Trying out API examples
+
+All API endpoints in this document have `Try it out` button which should produce `curl` command that can be use to
```
Comment (adinapoli-iohk) [inline wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs]: @akegalj There are some typos here ("have _a_ 'Try it out',  can be _used_ to etc). Would you mind proof-reading? Maybe Niamh, our technical writer, can help?

Comment (akegalj) [reply wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs]: Uh, yes - I am not native speaker :/ and my grammar sucks. Let me fix those two points and I will invite Niamh to proof read

Comment (akegalj) [reply wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs]: ok, I have contacted Niamh. Ill ping you when this is ready.

Comment (akegalj) [reply wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs]: Correction from Niamh https://docs.google.com/document/d/1DHBUV0rvtB0vErqLWVmEl3W6bqXWrgG4f9rAU72Qn_U/edit?usp=sharing

**Pull request merged**